src/main/webapp/appscript.js

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+  * Function that fetches vendor IDs from VendorServlet and adds them to the
+  * Customer ID dropdown.
+  */
+async function populateCustomerList() {
+  const response = await fetch('/VendorServlet');
+
+  // response is a json array that is parsed into json.
+  var customer_json = await response.json();
+
+  // Loops over this array to make the options for the Customer ID dropdown,
+  var html = '';
+  for (var i = 0; i < customer_json.length; i++) {
+    html += '<option value="' + customer_json[i] + '">'
```

Comment:
optional: make a small method for formatting this string? It would accept a string and return <option value="{input}>{input}</option>